common/address.go

```diff
@@ -118,3 +105,24 @@ func AddressFromBase58(encoded string) (Address, error) {
 
 	return ph, nil
 }
+
+func VerifyBase58(encoded string) ([]byte, error) {
```

Comment:
the original `AddressFromBase58` has verified this, so any reason you need this function? and this implementation is buggy since the checksum is not verified.

Reply:
sorry, what's the meaning of 'buggy'